lib/flop.ex

```diff
@@ -1906,11 +1906,16 @@ defmodule Flop do
   Updates the `order_by` and `order_directions` values of a `Flop` struct.
 
   - If the field is not in the current `order_by` value, it will be prepended to
-    the list. The order direction for the field will be set to `:asc`.
+    the list. By default, the order direction for the field will be set to
+    `:asc`.
   - If the field is already at the front of the `order_by` list, the order
     direction will be reversed.
   - If the field is already in the list, but not at the front, it will be moved
-    to the front and the order direction will be set to `:asc`.
+    to the front and the order direction will be set to `:asc` (or the custom
+    asc direction supplied in the `:directions` option).
+  - If the `:directions` option——a 2-element tuple——is passed, the first and
+    second elements will be used as custom sort declarations for ascending and
+    descending, respectively.
```

Comment:
@woylie I wasn't sure whether to document `:directions` here, or more explicitly in their own section (after the bits about the possible passage of a string as the second arg). My gut said that it makes more sense here than below, as the default order values were already being mentioned.